ipwb/replay.py

```diff
@@ -255,13 +255,33 @@ def show_uri(path, datetime=None):
 
     for idx, hLine in enumerate(hLines):
         k, v = hLine.split(': ', 1)
+
+        if k.lower() == 'transfer-encoding' and v.lower() == 'chunked':
+            resp.set_data(extractResponseFromChunkedData(payload))
         if k.lower() != "content-type":
             k = "X-Archive-Orig-" + k
+
         resp.headers[k] = v
 
     return resp
 
 
+def extractResponseFromChunkedData(data):
+    chunkDescriptor = -1
+    retStr = ''
+
+    (chunkDescriptor, rest) = data.split('\n', 1)
+    while chunkDescriptor.strip() != '0':
+        chunkDecFromHex = int(chunkDescriptor, 16)  # Get dec for slice
```

Comment:
Alternatively, we can check if ';' is present in the line. If so, split on it then use the first value as the chunk size. Casting this to an `int` instead of stripping would be better as well -- I'll give you that.